compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/BIRGen.java

```diff
@@ -460,6 +461,7 @@ public void visit(BLangFunction astFunc) {
         // Special %0 location for storing return values
         birFunc.returnVariable = new BIRVariableDcl(astFunc.pos, astFunc.symbol.type.getReturnType(),
                 this.env.nextLocalVarId(names), VarScope.FUNCTION, VarKind.RETURN, null);
+        birFunc.localVars.add(0, birFunc.returnVariable);
```

Comment:
This `add(0,` is used in multiple places. A new reader might not know what `0` signifies. If we define a constant for `0` it would add clarity.